Source/WebInspectorUI/UserInterface/Views/DetailsSection.css

```diff
@@ -252,12 +252,12 @@ body[dir=rtl] .details-section > .header::before {
     margin-inline-start: 5px;
 }
 
-.details-section > .content > .group > .row:is(.simple, .font-variation):has(.warning) {
+.details-section > .content > .group > .row.simple:has(.warning) {
```

Comment:
This basically restores `DetailsSection.css` to how it looked like before implementing the recent Fonts sidebar panel input controls.

@dcrousso I know you noted that this is a layering violation and it should live in its own stylesheet, perhaps `DetailsSectionSimpleRow.css`. I'll do that in a separate patch. This one is already sprawling.